torch/_dynamo/variables/builder.py

```diff
@@ -695,8 +695,14 @@ def build_key_value(k, v):
             istype(value, (types.ModuleType, replay_record.DummyModule))
             # type(torch.backends.cudnn) -> <class 'torch.backends.cudnn.CudnnModule'>
             # type(torch.ops) -> <class 'torch._ops._Ops'>
-            or value in [torch.backends.cudnn, torch.ops]
```

Comment:
If this isn't working, wouldn't:
```suggestion
            or any(value is obj for obj in (torch.backends.cudnn, torch.ops))
```
work?

Reply:
It works, thanks!